server/authorizer/authorizer.go

```diff
@@ -277,3 +282,20 @@ func GetBodyOnContext(ctx context.Context) (interface{}, bool) {
 	}
 	return input.parsed, true
 }
+
+// Note(philipc): Copied over from server/server.go
+func readPlainBody(r *http.Request) (io.ReadCloser, error) {
```

Comment:
Perhaps worth putting into some utils or internal/utils, whatever we've got at hand?

Reply:
Done. It's moved over the `util` in the latest changes.